gwpy/signal/qtransform.py

```diff
@@ -189,6 +189,8 @@ def transform(self, fseries, **kwargs):
         QPlane.transform
             compute the Q-transform over a single time-frequency plane
         """
+        if not numpy.isfinite(fseries).all():
```

Comment:
I think it is more intuitive to have a positive statement here:
```suggestion
        if numpy.isinf(fseries).any():
```

Reply:
@duncanmmacleod, I would agree, however `numpy.isinf` doesn't account for NaNs, which is the most common corner case this block is trying to catch.